.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,54 +1,25 @@
----
-name: Bug report
```

Comment:
These header sections are largely redundant information already contained in the issue metadata. Name is the title, about is the description, title is the title, so on.